csi.proto

```diff
@@ -947,6 +947,39 @@ message GetCapacityResponse {
   // storage. This field is REQUIRED.
   // The value of this field MUST NOT be negative.
   int64 available_capacity = 1;
+
+  // The largest size that may be used in a
```

Comment:
For volume plugins that don't need to deal with fragmentation issues, I think defining a stricter available_capacity would be beneficial to help us do batch processing of multiple volumes. Right now, we're limited to processing a single volume at a time, which is a major issue for supporting a single workload that requests multiple volumes.

Reply:
Improving the definition of `available_capacity` was intentionally left out because it seemed to need further discussion.

Reply:
I meant should we define a new available_capacity field that has a more strict definition?

Reply:
Yes, probably. But not in this PR :sweat_smile: 

What you want is tracked in https://github.com/container-storage-interface/spec/issues/301

Reply:
Not quite. So we said available_capacity could be interpreted 2 ways: 1) available capacity to provision a single volume and 2) total available capacity as reported by the storage system. Why do we only want to clarify one of the methods that this could be interpreted?

Reply:
In the last CSI community meeting we talked about whether `available_capacity` should be clarified and concluded that the wording should be left as-is until there is a specific use-case that needs a stricter definition than the one that is currently in the spec. See (or rather, listen): https://www.youtube.com/watch?list=PLypWp79Fch09Sv7Vo-mHgEo20cGU7oNBc&t=1917&v=ZB0Y05jo7-M&feature=youtu.be

Both @bswartz and @saad-ali confirmed.